guides/common/modules/proc_configuring-satellite-to-synchronize-content-with-a-local-cdn-server.adoc

```diff
@@ -11,7 +11,7 @@ Next, you must configure {ProjectServer} to synchronize content with the local C
 . In the upper left of the window, click *Downloads* and select *{ProjectName}*.
 . Click the *Content ISOs* tab.
 This page lists all the products that are available in your subscription.
-. Click the link for the product name, such as *Red Hat Enterprise Linux 7 Server (x86_64)* to download the ISO image.
+. Click the link for the product name, such as *{RHEL} 7 Server (x86_64)* to download the ISO image.
```

Comment:
I think this refers to the label on the portal where it is specifically in this server. If you have something else (like RHEL Server 7 instead of RHEL 7 Server) it may be confusing for users.